_source/_posts/2021-01-27-okta-ansible.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blog_post
+title: "Tutorial: Ansible and Account Automation with Okta"
+author: frederico-hakamine
+by: internal-contributor
+communities: [devops]
+description: "How to integrate Okta and Ansible to manage accounts in servers while abstracting account management from your playbooks"
+tags: [devops]
+tweets:
+- "Learn how to integrate Okta and Ansible to manage server accounts and abstract account management from your playbooks"
+- "Remove account management from your Ansible playbooks. Here's how"
+image: blog/
+type: conversion
+---
+
+**Tip:** This tutorial is part of our series on how to integrate Okta with popular Infrastructure as a Code solution. If you're not into Ansible, check out our [Puppet](/blog/2021/01/22/okta-puppet) and [Terraform](/blog/2020/04/24/okta-terraform-automate-identity-and-infrastructure) tutorials.
```

Comment:
Can we take this line out for now? Maybe comment it out so we can put it back in after the other posts are up?

Reply:
By the time Ansible goes out, Puppet and Terraform will be already out